src/renderer/components/LoginComponent/LoginComponent.scss

```diff
@@ -0,0 +1,46 @@
+
```

Comment:
Could you format this file? Just simple CSS formatting default to your IDE should work.